flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -102,14 +102,18 @@ public FlinkCatalog(
       String defaultDatabase,
       Namespace baseNamespace,
       CatalogLoader catalogLoader,
-      boolean cacheEnabled) {
+      boolean cacheEnabled,
+      long cacheExpirationIntervalMs) {
     super(catalogName, defaultDatabase);
     this.catalogLoader = catalogLoader;
     this.baseNamespace = baseNamespace;
     this.cacheEnabled = cacheEnabled;
 
     Catalog originalCatalog = catalogLoader.loadCatalog();
-    icebergCatalog = cacheEnabled ? CachingCatalog.wrap(originalCatalog) : originalCatalog;
+    icebergCatalog =
+        cacheEnabled
```

Comment:
Can we drop `cacheEnabled` and just use `cacheExpirationIntervalMs`?  When `cacheExpirationIntervalMs` is zero, it is disabled.

Reply:
thanks for pointing, address it.

Reply:
this breaks the compatibility. I feel the benefit probably doesn't justify the breakage. we should keep the boolean config and just support the additional expire interval.